content/documentation/zooming-images.md

```diff
@@ -0,0 +1,207 @@
+
+- When quantifying IIIF images, it’s hard to say how many IIIF images will be too many for a system to handle, because the number of individual files associated with a given IIIF image is highly variable.
+
+    **For example:**
+
+    4 levels of zoom ≈ 100 image tiles<br />
+    5 levels of zoom ≈ 329 image tiles<br />
+    6 levels of zoom ≈ 1,170 image tiles
+
+{{< /q-class >}}
+
+For hosting the image tiles, of you have institutional support, your digital department should be able to provide a solution. If you’re on your own, there are any number of options. You might check out [Amazon S3](https://aws.amazon.com/s3/), which is very performant and self-serve but requires some techinical savvy to get set up, or an independent hosting service like [Reclaim Hosting](https://reclaimhosting.com/shared-hosting/), which caters to the academic sector.
+
+## Display IIIF Images in Your Project
+
+To display a IIIF image in your project, you need to point to a IIIF `info.json` file for the image. Either one you’ve identified from an [external source](#use-existing-iiif-images), or one you’ve [created yourself](#creating-your-own-iiif-images) with `quire process --iiif`. If it is an image you processed yourself and it is still in the `processed/` directory, the path to the JSON file would be `/img/iiif/processed/FILENAME/info.json`.
+
+Along with the path to the `info.json` file, you also need to include `media_type: "iiif"` and a path to a lower-res version of the image, hosted in your project’s `static/img` directory (which again is used for Quire’s PDF and e-book outputs). All of this goes in your `figures.yml` file.
+
+```yaml
+- id: "irises"
+  src: "figures/irises.jpg"
+  media_type: "iiif"
+  iiif: "https://data.getty.edu/museum/api/iiif/671108/info.json"
+```
+
+The image can then be added to your Markdown files using the `q-figure-zoom` shortcode. This will display the static image on the page, and then when clicked, will open the figure viewer for the fully zoomable IIIF version.
+
+```go
+{{</* q-figure-zoom id="irises" */>}}
+```
+
+IIIF Images can also be displayed in the built-in image viewer on [entry pages](/documentation/pages). Do this by including the appropriate `id` in the [object data](/documentation/collection-catalogues/#capture-object-data) in your `objects.yml` file.
+
+```yaml
+object_list:
+  - id: 1
+    title: *Irises*
+    artist: Vincent van Gogh
+    year: 1889
+    medium: Oil on canvas
+    dimensions: 74.3 × 94.3 cm (29 1/4 × 37 1/8 in.)
+    location: J. Paul Getty Museum
+    link: https://www.getty.edu/art/collection/objects/826/vincent-van-gogh-irises-dutch-1889/
+    figure:
+      - id: "irises"
+```
+
+### Include a Static Fallback Image
+
+The static fallback image is required for displaying a version of the IIIF image outside of the zooming image viewer on the online version of your project, as well as for Quire’s PDF and e-book outputs. This is the standard `src` attribute in your `figures.yml` listing.
```

Comment:
Is this the same as the lower-res version mentioned in line 154? Do we need to mention it twice? It seems redundant. If we do want it twice can we consistently refer to it as a "static fallback image."

Reply:
Yes, it's a little redundant, but I wanted to hit it home a little, and also made sense to me to mention it first when I showed the yaml sample. But I'll make the name consistent as you suggest, and also add a link to the lower section. Thanks.